src/common_format.rs

```diff
@@ -173,6 +173,12 @@ pub fn manage_calendars<H>(file_handler: &mut H, collections: &mut Collections)
 where
     for<'a> &'a mut H: FileHandler,
 {
+    let calendar_exists =
+        file_handler.is_file("calendar_dates.txt") | file_handler.is_file("calendar.txt");
```

Comment:
what I don't like it's that `get_file_if_exists` already tests the existence of the file

Reply:
OR operator changed to logical one

For is_file instead of get_file_if_exists it's in debate

Reply:
method is_file makes sense because we don't need to open the file to check if it exists

Reply:
Can be used in any function having a file_handler in parameter.
But yes, only for the manage_calendars function for the moment.

Reply:
I prefer @datanel solution, not really because it removes a method in the trait, but because of race condition (there is calendars.txt when we check the existance of the file, but it is deleted before we open it).